README.md

```diff
@@ -28,10 +28,10 @@ git_repository(
 
 load(
   "@io_bazel_rules_docker//docker:docker.bzl",
```

Comment:
We should load this from `container:container.bzl` if we're going to use this name (which we should!).

Reply:
Well spotted. Thanks and fixed.